lib/smart_proxy_openscap/store_arf_reports.rb

```diff
@@ -0,0 +1,20 @@
+require 'smart_proxy_openscap/store_arf_fs'
+module Proxy::OpenSCAP
+  class StoreArfReports < StoreArfFS
+    def store(cn, id, date, data)
+      begin
+        target_dir = create_directory(Proxy::OpenSCAP::Plugin.settings.reportsdir, cn, id, date)
+      rescue StandardError => e
+        raise Proxy::OpenSCAP::StoreReportError.new "Could not fulfill request: #{e.message}"
```

Comment:
raise Proxy::OpenSCAP::StoreReportError, "Could not fulfill request: #{e.message}"